ts/features/itwallet/common/utils/itwClaimsUtils.ts

```diff
@@ -27,29 +25,24 @@ import { ParsedCredential, StoredCredential } from "./itwTypesUtils";
  */
 
 /**
- * Retrieves the organization name from the evidence claim of the given credential.
- * If the evidence claim is not present or cannot be decoded, a fallback value is returned.
- * @param credential - The parsed credential object.
- * @returns The organization name from the evidence claim or a fallback value.
+ * We strongly discurage direct claim manipulation, but some spacial cased must be addressed with direct access
```

Comment:
Small typo: `discurage`!

Reply:
Addressed in https://github.com/pagopa/io-app/pull/6074/commits/b07750f59059602857b02be7907fadd88ca2c35f and https://github.com/pagopa/io-app/pull/6074/commits/f4386520338ea40306c1f9363ad4268a4fbf508f